Makefile

```diff
@@ -71,6 +71,18 @@ web:
 		rm $$touched; \
 	done
 
+hinted:
+	mkdir -p out/hinted
+	for source in src/hinted/*.ttf; do \
+		basename=$$(basename $$source); \
+		case $$source in \
+			src/hinted/Roboto-*) unhinted=out/RobotoTTF/$$basename ;; \
+			*) unhinted=out/RobotoCondensedTTF/$$basename ;; \
+		esac; \
+		final=out/hinted/$$(basename $$source); \
```

Comment:
no need to get `$$(basename $$source)` again, you already saved that into `$$basename` at the beginning of the loop.